opencti-platform/opencti-front/lang/front/fr.json

```diff
@@ -950,7 +951,7 @@
   "Enrich": "Enrichir",
   "Enrichment": "Enrichissement",
   "Enrichment connectors": "Connecteurs d'enrichissement",
-  "Enroll in playbook": "S'inscrire au cahier de jeu",
+  "Enroll in playbook": "Envoie vers un playbook",
```

Comment:
weird yo have it " conjugué alors que les autres verbes sont à l'infinitif":
![image](https://github.com/user-attachments/assets/01c19c57-d4be-44f0-b8be-cad9342e1010)

Reply:
done